src/SMU-Agilent_B29xx/main.py

```diff
@@ -155,8 +162,8 @@ def configure(self):
         if self.source.startswith("Voltage"):
             self.port.write(":SOUR%s:FUNC:MODE VOLT" % self.channel)
             # sourcemode = Voltage
-            self.port.write(":SOUR%s:VOLT:MODE FIX" % self.channel)
-            # sourcemode fix
+            self.port.write(":SOUR%s:VOLT:MODE SWE" % self.channel)
```

Comment:
Would it make sense to use the Sweep mode only for the pulse while keeping it FIX for non-pulse measurements?

Reply:
It would give me a better feeling if the standard DC measurement runs with FIX and the new pulse mode can run with SWE. I can help you with implementing it. Just let me know. Basically, I would shift this line to where the cases are for DC and pulse handling. In "apply" phase I would also add a section to handle DC and pulse mode.